src/08-tasks/tasks.apib

```diff
@@ -12,6 +12,7 @@ Get a list of tasks.
             + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional, nullable) - Returns tasks that are assigned to this user or to a team to which this user belongs. When passing `null`, it returns tasks that are unassigned.
             + milestone_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, optional)
             + completed: true (boolean, optional)
+            + scheduled: true (boolean, optional)
```

Comment:
Q: Can completed & scheduled occur at the same time? If not should we make this a "status" property instead?

Reply:
Yes, you can complete a task without scheduling it.